src/scss/themes.scss

```diff
@@ -31,6 +31,13 @@ html {
     #app {
       color: #2c3e50;
     }
+    #chatter-threshold {
+      background-color: #EEEEEE;
+      border:none;
+    }
+    .chatter-detected {
+      background-color: rgb(233, 107, 107) !important;
```

Comment:
why important?

Reply:
can do a bit of refactor on it, but it's due to `detected` class and i'm not sure of how the output of vue is done for the CSS and how the browser will prioritize the stylesheet rules.

Reply:
![2019-08-14 08_37_11-QMK Configurator](https://user-images.githubusercontent.com/6959636/62999590-39b01880-be6f-11e9-8c97-4701c93be980.png)

got to use important here

Reply:
@noroadsleft  any suggestions?

Reply:
usually more specific wins

Reply:
or i can do : 
```
.tester-key.detected.chatter-detected,
.tester-key.chatter-detected {
...
```

Reply:
> or i can do :
> 
> ```
> .tester-key.detected.chatter-detected,
> .tester-key.chatter-detected {
> ...
> ```

I like this better.

Everything else looks good.